### ISSUE TITLE
Home and Events pages should have H1s

### ISSUE THREAD

**Issue by thejkayway** [thejkayway/m4apdx#10]
The home page and events pages should have H1s.

- The events page is one of the dynamically generated pages. It gets generated by gatsby at build time in `src/gatsby/node/createPages.js`. This happens by filling the template `src/templates/events.js` with data from contentful.
  - We have an existing styled component for h1 page titles (`src/components/PageTitle.js`). We can add this component to the `<Container>` component inside `events.js` as its first child.

- Content for the home page is entirely defined in contentful, adding a markdown h1 (e.g. `# Example Header`) there will create an html h1 onsite.
  - If we want to take away content editors' abilities to remove or change this we could modify `src/templates/page.js`, maybe to conditionally render a title pulled from contentful instead, but that sounds like a later thing to me
